### PR TITLE
Leave Request UI

### PR DESCRIPTION
**Pull request by tlan4**
Integrated leave request's UI into the employee view

### CODE REVIEW

**Review by lungerjo — COMMENTED** [review]
Great looking UI.

**Review by jianwang0722 — APPROVED** [review]
modifications made look correct and changes made adhere to the builder design pattern. In the leaveRequestUI file, the part that initialize the use case classes and the data access is commented out because the implementation of the data access is not yet complete.

File: src/main/java/controller/LeaveRequestController.java [hunk 1]
```diff
@@ -9,13 +9,13 @@
...
 
 public class LeaveRequestController {
-    private final LeaveRequestInputBoundary userInput;
+    final LeaveRequestInputBoundary userInput;
 
     public LeaveRequestController(LeaveRequestInputBoundary userInput) {
         this.userInput = userInput;
     }
 
-    LeaveRequestResponseModel create(User user, java.lang.String message, LeaveType leaveType,
```
Comment (zeyuzhang516) [inline src/main/java/controller/LeaveRequestController.java:18]: Could you briefly explain why you change the type of the message from String to java.lang.String?

**Review by zeyuzhang516 — APPROVED** [review]

File: src/main/java/controller/LeaveRequestController.java (hunk 1 above)
Comment (tlan4) [reply src/main/java/controller/LeaveRequestController.java:18]: I  messed up my entities and InteliiJ automatically changed all of it to java.lang.String, I am still trying to find all it and change it back to String. I don't think there is any difference between them.

**Review by GitHaoBao — APPROVED** [review]
All modifications look good. Thanks for providing an example for us to learn how to connect the controller and the ui part.

**Pull request merged**